modules/fb_instant_articles_rss/fb_instant_articles_rss.module

```diff
@@ -0,0 +1,92 @@
+<?php
+
+/**
+ * @file
+ * Facebook Instant Articles RSS module.
+ */
+
+/**
+ * Implements hook_menu().
+ */
+function fb_instant_articles_rss_menu() {
+  $items['fbinstant.rss'] = array(
+    'description' => 'Facebook Instant Articles RSS endpoint',
+    'page callback' => 'fb_instant_articles_rss_page',
+    'file' => 'includes/rss.inc',
+    'access arguments' => array('access content'),
+  );
+
+  $items['admin/config/services/fb-instant-articles/rss'] = array(
+    'title' => 'Facebook Instant Articles RSS',
+    'description' => 'RSS settings for Facebook Instant Articles.',
+    'page callback' => 'drupal_get_form',
+    'page arguments' => array('fb_instant_articles_rss_settings'),
+    'access arguments' => array('administer fb_instant_articles_rss'),
+    'file' => 'includes/admin.inc',
+  );
+
+  return $items;
+}
+
+/**
+ * Implements hook_permission().
+ */
+function fb_instant_articles_rss_permission() {
+  $permissions = array();
+
+  $permissions['administer fb_instant_articles_rss'] = array(
+    'title' => t('Administer Facebook Instant Articles RSS'),
+  );
+  return $permissions;
+}
+
+/**
+ * Implements hook_node_operations().
+ */
+function fb_instant_articles_rss_node_operations() {
```

Comment:
You're right.  As I noted there, I had it in my head that the views integration used this data in a custom filter, which it currently does not, so all good.